### ISSUE TITLE
[compatibility] No Chat Reports

### ISSUE THREAD

**Issue by Lucky-56** [txnimc/PronounMC#1]
currently having both mods installed doesn't work as [No Chat Reports](https://modrinth.com/mod/no-chat-reports) removes all ties to the Microsoft Account

**Comment by CherrieTheShifter** [comment]
Yeah, I have this same issue even now. Is it possible to fix this, or do we gotta delete that mod or???
Like will No Chat Reports, ps I use the ncr mod but from curseforge, i hope so idk. cuz this is still an issue!.
